lib/restforce/sobject.rb

```diff
@@ -16,6 +16,12 @@ def describe_layouts(layout_id = nil)
       @client.describe_layouts(sobject_type, layout_id)
     end
 
+    def refresh
+      ensure_id
+      new_record = @client.find(sobject_type, self.Id)
```

Comment:
Are we confident that `#find` works with all `sobject_type`s? Is there anything we could do to validate that?

Reply:
I haven't found any object which it doesn't—is there a specific case you have in mind where you think we'd run into trouble?

Reply:
I don't have any specific ideas, to. E honest. I'm happy to run with this, and I'm sure people will tell us if it doesn't work across the board. 

If it doesn't, you should get a sensible exception anyway.

Reply:
Got it! Will change.

What do you think about aliasing `refresh` to the `reload` method?

Reply:
❤️ I'd prefer just to have one name to keep the surface area small. We can use both words when we document it in the readme to make it discoverable.